docs/5_2_scheduled_execution_api.adoc

```diff
@@ -8,379 +8,125 @@ The simulation algorithm signals to the FMU that it supports and has recognized
-Calls are based on ticks of <<periodic>> or <<periodic,aperiodic>> <<inputClock,`input clocks`>>.
-These <<inputClock,`input clock`>> ticks can be based on <<clock>> ticks from FMU external sources (e.g. <<outputClock,`output clocks`>> of other FMUs).
-The <<inputClock>> ticks can also be based on <<outputClock,`output clock`>> ticks of the same FMU.
-Refer to <<math-scheduled-execution>> and <<CommunicationPointClocks>>.
-The <<fmi3ActivateModelPartition>> function is not called for <<outputClock,`output clocks`>> of an FMU.
+* latexmath:[(\mathbf{y}_\mathit{d,k_i}, \mathbf{x}_\mathit{d,k_i}, \mathbf{w}_\mathit{d,k_i}) := \mathbf{f}_{\mathit{activate}}({}^{\bullet}\mathbf{x}_\mathit{d}, \mathbf{w}_d, \mathbf{u}_\mathit{d}, \mathbf{p}, \mathbf{k_i})] +
+* Update previous values of discrete states of the corresponding model partition: latexmath:[{}^\bullet\mathbf{x}_\mathit{d,k_i}:=\mathbf{x}_\mathit{d,k_i}].
+|<<fmi3ActivateModelPartition>>
 
-Note that this is a different timing concept compared to <<fmi3DoStep>> calls for Co-Simulation FMUs or the <<fmi3SetTime>> for Model Exchange FMUs.
-A <<fmi3ActivateModelPartition>> call will compute the results of the model partition defined by <<clockReference>> and <<clockElementIndex>> (i.e. <<valueReference>> of the variable that defines a <<clock>> and an element index into that for arrays) for the current <<clock>> tick latexmath:[t_i].
+|latexmath:[(\mathbf{v}_{u}) := \mathbf{f}_{\mathit{inter}}({}^{\bullet}\mathbf{x}_d, \mathbf{u}_{d}, {}^{\bullet}\mathbf{u}_{u}, \mathbf{p} )]
+|<<fmi3CallbackIntermediateUpdateSE>>
 
-If required, the FMU can internally derive the <<clock>> interval latexmath:[\Delta T_i] based on the last <<clock>> tick time latexmath:[t_{i-1}] i.e. last `activationTime` for this <<clockReference>> and <<clockElementIndex>> (based on last <<fmi3ActivateModelPartition>> call).
+|====
 
-Consecutive calls to <<fmi3ActivateModelPartition>> for a <<clockReference>> and <<clockElementIndex>> (i.e. <<valueReference>> of <<clock>> variable and element index into that for arrays) must have strictly monotonically increasing `activationTime` latexmath:[t_i].
+Allowed Function Calls::
 
 Function <<get-and-set-variable-values,`fmi3Set{VariableType}`>>::
 This function can be called before scheduling a model partition for variables assigned to that model partition via its associated <<clock>> and all variables not associated to a <<clock>> (global variables).
 
-Function <<get-and-set-variable-values,`fmi3Get{VariableType}>>`, `fmi3GetOutputDerivatives`, <<fmi3GetDirectionalDerivative>>::
+Function <<get-and-set-variable-values,`fmi3Get{VariableType}>>`::
 These functions can be called after the computation of a model partition for variables assigned to that model partition via its associated <<clock>> and all variables not associated to a <<clock>> (global variables).
 
++
 Set/get operations must be atomic for a single variable.
```

Comment:
What does "atomic" mean here?

Reply:
This is comp-sci lingo for "uninteruptable" and needed here because SE also allows for preemtion.
I will close this because this should be clear to anyone implementing this (and `operation` and `atomic` are used in the same sentence and google produces a good wiki article).
Please propose improvement if you reopen.